Codeinwp/neve
# Feat/edd compatibility

### Summary

Adds basic EDD support to free theme. Still needs work. Opening pull request for feedback

### Will affect visual aspect of the product

YES

### Screenshots 

https://vertis.d.pr/XdIWdL
https://vertis.d.pr/fdlmbd

### Test instructions


- Install EDD
- Install Theme
- Test Archive and single downloads should be better formatted


Closes https://github.com/Codeinwp/neve-pro-addon/issues/972
